src/discordtipbot/discordbot.cr

```diff
@@ -533,6 +535,44 @@ class DiscordBot
     end
   end
 
+  def glucky(msg : Discord::Message, cmd_string : String)
+    return reply(msg, "**ERROR**: This command doesn't work in DMs") if private?(msg)
+
+    cmd_usage = "#{@config.prefix}lucky [amount]"
+
+    # cmd[0]: command, cmd[1]: amount, cmd[2]: people"
+    cmd = cmd_string.split(" ")
+
+    return reply(msg, cmd_usage) unless cmd.size > 2
+
+    amount = amount(msg, cmd[1])
+    return reply(msg, "**ERROR**: You have to specify an amount! #{cmd_usage}") unless amount
+
+    people = cmd[2].to_i? # Number of people to give money to.
+    return reply(msg, "**ERROR**: The number of people should be a number. obviously.") unless people
+
+    users = active_users(msg)
+
+    amount_each = amount/people
+
+    return reply(msg, "**ERROR**: There arent that many people!") unless users && (users = users.to_a).size > people
```

Comment:
This breaks if you want to glucky on the same amount of people who are possible to get lucky'd

Reply:
You still haven't fixed this

Reply:
If there are only two people active, shouldn't you just use rain?

Reply:
Not really. Just use `>=` instead of `>`

Reply:
mk.

Reply:
amended to last commit